test/pageobjects/ProductDetailsPage.js

```diff
@@ -1,10 +1,34 @@
 const {PageObject} = require("./PageObject");
 
-class ProductDetailed extends PageObject {
+class ProductDetails extends PageObject {
+
+    get url() {
+        return '/inventory-item.html?id=4'
+    }
+
+    open() {
+        return super.open(this.url);
+    }
+
+    get btnBackToProducts() {
+        return browser.$("//button[@id='back-to-products']"); //
+    }
 
     get detailsImage() {
         return browser.$("//img[@class='inventory_details_img']");
     }
+
+    get detailsName() {
+        return browser.$("//div[@class='inventory_details_name large_size']");
+    }
+
+    get detailsDescription() {
+        return browser.$("//div[@class='inventory_details_desc large_size']");
```

Comment:
same as above

Reply:
the xpath has been changed into "//*[contains(@class,'inventory_details_desc ')]"